baselines/splunk_command_and_scripting_interpreter_risky_spl_mltk_baseline.yml

```diff
@@ -0,0 +1,85 @@
+name: Splunk Command and Scripting Interpreter Risky SPL MLTK Baseline
+id: 273df2f7-643a-451a-8d4d-637e39eadc87
+version: 1
+date: '2022-05-27'
+author: Abhinav Mishra, Kumar Sharad and Xiao Lin, Splunk
+type: Baseline
+datamodel:
+- Splunk_Audit
+description: 'This search is to build a per user command and scripting risky SPL detection model using MLTK 
+  DensityFunction algorithm based on audit log data of Splunk apps. The model is built from past 7 days users 
+  history of running search commands from 
+  https://docs.splunk.com/Documentation/Splunk/latest/Security/SPLsafeguards#Commands_that_trigger_the_warninga, 
+  and aggregate the total search run time for each hour as indicator of user behavior. The default threshold of 
+  outlier boundary is defined as 0.1%, which means that the possibility of search run time exceeding that 
+  boundary is smaller than 0.1% (three 9s - 0.999), indicating a risky use of these commands. Users can adjust 
+  this threshold 0.1% to higher or lower value depending on desired false positive report rate. This search should 
+  be scheduled to run at least as frequently as every 7 days. The name of machine learning model generated is 
+  "risky_command_abuse" and should be configured to be globally shared (not private) in MLTK app as documented here:
+  https://docs.splunk.com/Documentation/MLApp/5.3.1/User/Models#Sharing_models_from_other_Splunk_apps
+  unless the same account of training this model will be used to perform inference using this model for anomaly 
+  detection. '
+search: '| tstats values(Search_Activity.user) as user, sum(Search_Activity.total_run_time) as run_time count 
+  FROM datamodel=Splunk_Audit.Search_Activity WHERE (Search_Activity.user!="") 
+  AND (Search_Activity.total_run_time>1) AND (earliest=-7d@d latest=now) 
+  AND (Search_Activity.search IN ("*| runshellscript *", "*| collect *","*| delete *", "*| fit *", "*| outputcsv *", 
```

Comment:
While I don't disagree with including the | in the filther, this does limit the risky commands to be exactly one space after the |. It might be worth calling this out.

Reply:
added note in how_to_implement